usr/sbin/so-netsniff-ng-cron

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+# Define initial variables
+SENSORTAB="/etc/nsm/sensortab"
+INTERFACES=$(grep -v '#' $SENSORTAB | awk '{print $1}')
+NETSNIFF_ENABLED="no"
+NETSNIF_RUNNING="no"
+OS_DATE=$(date +"%Y-%m-%d")
+
+# Check to see if netsniff-ng is enabled.
+for i in $INTERFACES; do
+	if grep -q 'PCAP_ENABLED="yes"' /etc/nsm/$i/sensor.conf; then
+		NETSNIFF_ENABLED="yes"
+	fi
+done
+
+# Check to see if netsniff-ng is running.
+if NETSNIFF_ENABLED="yes"; then
+	if pgrep -af netsniff-ng > /dev/null 2>&1; then
+		NETSNIFF_RUNNING="yes"
+	fi 
+fi
+
+# Check if netsniff-ng is 
+if NETSNIFF_RUNNING="yes"; then
+	NETSNIFF_DATE=$(pgrep -af -oldest netsniff-ng | cut -d'/' -f6)	
+fi
+
+# If netsniff-ng date and OS date are not equal, restart netsniff-ng
+if [ "$NETSNIFF_DATE" == $OS_DATE ]; then
+	exit
+else
+	nsm_sensor_ps-restart --only-pcap > /dev/null 2>&1 && echo "$(date) -  netsniff-ng was restarted due to a date mismatch (Current date is: $OS_DATE.  Previously writing to $NETSNIFF_DATE)." > /var/log/nsm/netsniff-sync.log
```

Comment:
If we do move this code inside the for-loop, then `nsm_sensor_ps-restart` should be executed with just that sensor name.

Also, when writing to `/var/log/nsm/netsniff-sync.log`, we probably want to append (`>>`) rather than overwrite (`>`).